packages/core/cli/src/scripts/createProject.ts

```diff
@@ -0,0 +1,79 @@
+const compileTemplate = require('@vue-storefront/nuxt-theme/scripts/compileTemplate');
+const getAllFilesFromDir = require('@vue-storefront/nuxt-theme/scripts/getAllFilesFromDir.js');
+const { copyFile } = require('@vue-storefront/nuxt-theme/scripts/copyThemeFiles');
+const consola = require('consola');
+const chalk = require('chalk');
+
+const fs = require('fs');
+const path = require('path');
+
+const log = {
+  info: (message) => consola.info(chalk.bold('VSF'), message),
+  success: (message) => consola.success(chalk.bold('VSF'), message),
+  warning: (message) => consola.warning(chalk.bold('VSF'), message),
+  error: (message) => consola.error(chalk.bold('VSF'), message)
+};
+
+const buildFileTargetPath = (file: string, targetPath: string, chopPhrase: string): string => targetPath + (file.replace(chopPhrase, ''));
+
+const copyThemeFiles = (filesDir: string | Array<string>, targetPath: string, chopPhrase: string) => {
+  if (fs.statSync(filesDir).isDirectory()) {
+    return Promise.all(getAllFilesFromDir(filesDir as Array<string>).map(
+      file => copyFile(file, buildFileTargetPath(file, targetPath, chopPhrase))
+    ));
+  } else {
+    return copyFile(filesDir, buildFileTargetPath(filesDir as string, targetPath, chopPhrase));
+  }
+};
+
+const removeDevMagicComment = (source: string): string => source.replace(/\s+(\/\/ @core-development-only-start)(.*?)(\/\/ @core-development-only-end)/sg, '');
+const removeCoreDevelopmentMode = (source: string): string => source.replace(/coreDevelopment:(\s)*?true[,]?/, '');
```

Comment:
*8suggestion** can be done with magic comment as well. By doing this you're keeping same convention for all things that should be removed in clients project